src/Core/Model.php

```diff
@@ -461,6 +461,16 @@ public static function getFields() {
     return self::getInstance()->_getFields();
   }
 
+  private function _getCaps() {
```

Comment:
Please rename this method to `_getCapabilities`, there's no reason to abbreviate.